dashboard/test/controllers/home_controller_test.rb

```diff
@@ -11,14 +11,24 @@ class HomeControllerTest < ActionController::TestCase
     Properties.stubs(:get).returns nil
   end
 
-  test "redirect index when signed in" do
+  test "redirect index when signed in (if not student with course progress)" do
```

Comment:
nit: reword something that seems more intentional, like `redirect signed in non-students with course progress to index`

Reply:
Though with similar trickiness with my above comment because it's not necessarily a "non-student", but rather a "non-(student with course progress)" :)

Reply:
Maybe `redirect to index when signed in and not a student with course progress` ?